pkgs/by-name/ry/rygel/package.nix

```diff
@@ -76,11 +81,13 @@ stdenv.mkDerivation (finalAttrs: {
       gtk3
```

Comment:
Actually, you still have GTK here.

Reply:
Nice catch, should have removed that.

Reply:
https://github.com/NixOS/nixpkgs/pull/362483

Reply:
I've opened #362484 to fix this.